python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -79,6 +87,10 @@ def start_ganga(gangadir_for_test='$HOME/gangadir_testing', extra_opts=[]):
         else:
             logger.info("InternalServices still running")
 
+        # The queues are shut down by the atexit handlers so we need to start them here
+        from Ganga.Core.GangaThread.WorkerThreads import startUpQueues
+        startUpQueues()
```

Comment:
Is this always needed? I forget where 'startUpQueues' is in the bootstrap code but I don't think the function prevents from the queues system being 'started' twice which may lead to threads being lost.
Is it worth adding a test within 'startUpQueues' for this?